groups/alteryx-sandbox/nlb.tf

```diff
@@ -0,0 +1,99 @@
+# Load Balancer
+resource "aws_lb" "alteryx_sandbox" {
+  name               = "alb-${var.application}-${var.application_environment}"
```

Comment:
If this is an NLB, the resource name prefix `alb-` should be updated to reflect this (or removed altogether).

Reply:
Have now updated with nlb- and replaced the alb.tf file for nlb.tf